snap/snapcraft.yaml

```diff
@@ -1,55 +1,58 @@
-name: srsran
+name: srslte
 base: core20
-version: '22.04.1'
+version: '20.10'
 summary: srsRAN is a 4G/5G software radio suite developed by SRS.
 description: srsRAN is a 4G/5G software radio suite developed by SRS.
-
 grade: devel
-confinement: devmode
-
+confinement: strict
 apps:
+  srsenb:
+    command: usr/bin/srsenb
+    environment:
+      LD_LIBRARY_PATH: $SNAP/usr/lib/x86_64-linux-gnu/
+  srsue:
+    command: usr/bin/srsue
+    environment:
+      LD_LIBRARY_PATH: $SNAP/usr/lib/x86_64-linux-gnu/
   srsepc:
-    command: usr/local/bin/srsepc
+    command: usr/bin/srsepc
     plugs:
       - home
       - network
       - network-bind
       - network-control
+    environment:
+      LD_LIBRARY_PATH: $SNAP/usr/lib/x86_64-linux-gnu/
+  install:
+    command: usr/bin/srslte_install_configs.sh
+    environment:
+      SRSRAN_INSTALL_DIR: $SNAP/usr/share/srslte/
 
+layout:
+  /usr/share/srslte:
+    bind: $SNAP/usr/share/srslte
 parts:
-
-  libzmq:
```

Comment:
Did you check if the snap actually works? I remember Javier had issues with ZeroMQ installed from deb rather than compiled.

Reply:
Yes, it works.
This version of the snap works good in `devmode` :)